kv/kvserver/intentresolver: TestIntentResolutionUnavailableRange failed

kv/kvserver/intentresolver.TestIntentResolutionUnavailableRange [failed](https://mesolite.cluster.engflow.com/invocations/default/e0b53ac7-5485-45c5-b748-2ef6b9a4ef9b?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cva3Yva3ZzZXJ2ZXIvaW50ZW50cmVzb2x2ZXI6aW50ZW50cmVzb2x2ZXJfdGVzdA==) on release-25.4 @ [a2df9f64b9bab37cf8695ab575d3260288ab5a66](https://github.com/cockroachdb/cockroach/commits/a2df9f64b9bab37cf8695ab575d3260288ab5a66):


```
=== RUN   TestIntentResolutionUnavailableRange
    test_log_scope.go:171: test logs captured to: outputs.zip/logTestIntentResolutionUnavailableRange2938239951
    test_log_scope.go:82: use -show-logs to present logs inline
    intent_resolver_integration_test.go:455: 
        	Error Trace:	pkg/kv/kvserver/intentresolver/intent_resolver_integration_test.go:455
        	Error:      	Received unexpected error:
        	            	pq: job 1131195708847554561 could not be loaded: failed to get progress for job 1131195708847554561: job-info-get: operation "intent_resolver_ir_batcher.sendBatch" timed out after 1.285s (given timeout 1s): result is ambiguous: after 1.28s of attempting command: context deadline exceeded
        	Test:       	TestIntentResolutionUnavailableRange
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestIntentResolutionUnavailableRange2938239951
--- FAIL: TestIntentResolutionUnavailableRange (72.35s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv-triage
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestIntentResolutionUnavailableRange.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-57635

--- a/pkg/kv/kvserver/intentresolver/intent_resolver_integration_test.go
+++ b/pkg/kv/kvserver/intentresolver/intent_resolver_integration_test.go
@@ -373,7 +373,7 @@ func TestIntentResolutionUnavailableRange(t *testing.T) {
 
 	// This test sometimes times out trying to replicate 3 tables t1, t2, t3 with
 	// replication factor 1 located on different servers under stress.
-	skip.UnderStress(t)
+	skip.UnderDuress(t)
 
 	ctx := context.Background()
 

